README.md

```diff
@@ -514,6 +514,9 @@ A curated list of awesome inclusive events/projects/initiatives for women in the
 
     List of initiatives and information about women in tech as well as the industry's gender gap.
 
+  - [awesome-diversity](https://github.com/folkswhocode/awesome-diversity)
+
+    A curated list of amazingly awesome articles, websites and resources about diversity in technology.
```

Comment:
Please add an "empty line" after the description. That's why you are getting the error on "build".

Also, please rebase with the latest master in order to avoid the order error.